data/adapters/gaf_adapter.py

```diff
@@ -0,0 +1,129 @@
+import os
+import gzip
+import json
+import hashlib
+
+from Bio.UniProt.GOA import gafiterator
+
+from adapters import Adapter
+from db.arango_db import ArangoDB
+
+# GAF files are defined here: https://geneontology.github.io/docs/go-annotation-file-gaf-format-2.0/
+#
+# Example:
+# !gaf-version: 2.2
+# !
+# !generated-by: GOC
+# !
+# !date-generated: 2023-04-02T12:17
+# !
+# ...
+# !
+# !=================================
+# !
+# !Documentation about this header can be found here: https://github.com/geneontology/go-site/blob/master/docs/gaf_validation.md
+# !
+# UniProtKB	A0A024RBG1	NUDT4B	enables	GO:0003723	GO_REF:0000043	IEA	UniProtKB-KW:KW-0694	F	Diphosphoinositol polyphosphate phosphohydrolase NUDT4B	NUDT4B	protein	taxon:9606	20230306	UniProt
+# UniProtKB	A0A024RBG1	NUDT4B	enables	GO:0046872	GO_REF:0000043	IEA	UniProtKB-KW:KW-0479	F	Diphosphoinositol polyphosphate phosphohydrolase NUDT4B	NUDT4B	protein	taxon:9606	20230306	UniProt
+# UniProtKB	A0A024RBG1	NUDT4B	located_in	GO:0005829	GO_REF:0000052	IDA		C	Diphosphoinositol polyphosphate phosphohydrolase NUDT4B	NUDT4B	protein	taxon:9606	20161204	HPA
+# UniProtKB	A0A075B6H7	IGKV3-7	involved_in	GO:0002250	GO_REF:0000043	IEA	UniProtKB-KW:KW-1064	P	Probable non-functional immunoglobulin kappa variable 3-7	IGKV3-7	protein	taxon:9606	20230306	UniProt
+# UniProtKB	A0A075B6H7	IGKV3-7	located_in	GO:0005886	GO_REF:0000044	IEA	UniProtKB-SubCell:SL-0039	C	Probable non-functional immunoglobulin kappa variable 3-7	IGKV3-7	protein	taxon:9606	20230306	UniProt
+
+
+class GAF(Adapter):
+    SKIP_BIOCYPHER = True
+    DATASET = 'gaf'
+    OUTPUT_PATH = './parsed-data'
+    RNACENTRAL_ID_MAPPING_PATH = './samples/rnacentral_ensembl_gencode.tsv.gz'
+    SOURCES = {
+        'human': 'http://geneontology.org/gene-associations/goa_human.gaf.gz',
+        'human_isoform': 'http://geneontology.org/gene-associations/goa_human_isoform.gaf.gz',
```

Comment:
I don't see where human_isoform source get used.

Reply:
you can also pass the isoform file to the adapter, example here: https://github.com/IGVF-DACC/igvf-catalog/pull/49/files#diff-1f685f4d2aa58625570e3d16f60f7e6c4eb75f97406c0f7421ea45b0ae1be0caR58